core/spec/models/spree/promotion_spec.rb

```diff
@@ -384,6 +384,73 @@
     end
   end
 
+  context '#unstarted?' do
+    it 'should not be unstarted if no starts_at date' do
+      promotion.starts_at = nil
+      expect(promotion.unstarted?).to eq(false)
```

Comment:
I find `.to be true` and `.to be false` more easy on the eyes

Reply:
there are `to be_truthy` and `to be_falsey` as well.